### DISCUSSION TITLE
Browser replacement doesn't specify for the main module. Why?

### DISCUSSION THREAD

**Discussion by BunnyBoss75** [aws/aws-sdk-js-v3#4606] [Q&A]
The package.json contains browser replacement for 'module' field (e.g. "./dist-es/runtimeConfig": "./dist-es/runtimeConfig.browser"), but doesn't specify for 'main' field. Needed replacement exists for need runtime.
Why it doesn't specify in package.json? Is there some reason?

**Comment by yenfryherrerafeliz** [comment]
Hi @BunnyBoss75, thanks for opening this discussion. I can not understand clearly your inquire, could you please elaborate more?, are you getting any specific errors, if so, could you please also provide logs, or is the current configuration affecting you somewhere, if so, could you please explain why.

Thanks!
